homework-assignments-1/task-2.js

```diff
@@ -0,0 +1,22 @@
+class User {
+    #password = "helloworld123";
```

Comment:
you shouldn't give default values for member properties. it must be initialized in constructor.